src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -36,6 +36,28 @@ export default class SimidMapCreative extends BaseSimidCreative {
      * @private {?string}
      */
     this.searchQuery_ = null;
+    /**
+     * The LatLng coordinates representing the closest advertised location.
+     * @private {?google.maps.LatLng}
+     */
+    this.activeLocation_ = null;
+    /**
+     * The LatLng coordinates representing the user's current location.
+     * @private {?google.maps.LatLng}
+     */
+    this.currentLocation_ = null;
+    /**
+     * The DirectionsRenderer object that displays directions from
+     * the given request.
+     * @private @const {!google.maps.DirectionsRenderer}
+     */
+    this.directionsRenderer_ = new google.maps.DirectionsRenderer();
+
+    /**
+     * A list of the travel modes supported by the Google Maps API.
+     * @private @const {!Array<!string>}
+     */
+    this.transportMethods_ = [("Driving"),("Walking"),("Bicycling"),("Transit")];
```

Comment:
This can be top-level, (so not a class variable, but a global one).

Also, no need to put each string in `()`

Reply:
fixed